### PR TITLE
docs: write contributing.md 

### LINKED ISSUE #191: [Feature]: Add CONTRIBUTING.md
### **Package Scope**

- [x] Add to an existing package

Package name: for all

### PR DESCRIPTION
**Pull request by okinawaa**
# Overview

fix #191 

Later, I'll do the translation work on another PR Let me know if there's anything missing!

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- _Don't_ restrict your search to only open issues. An issue with a title similar to yours may have been closed as a duplicate of one with a less-findable title.
+- Check for synonyms. For example, if your bug involves an interface, it likely also occurs with type aliases or classes.
+- Search for the title of the issue you're about to log. This sounds obvious but 80% of the time this is sufficient to find a duplicate when one exists.
+- Read more than the first page of results. Many bugs here use the same words so relevancy sorting is not particularly strong.
+- If you have a crash, search for the first few topmost function names shown in the call stack.
+
+## 2. Do you have a question?
+
+The issue tracker is for **issues**, in other words, bugs and suggestions.
+If you have a _question_, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/suspensive), your favorite search engine, or other resources.
```
Comment (okinawaa) [inline CONTRIBUTING.md:26]: I have a question here.
What's better  getting questions about libraries betweenstack overflow and GitHub issues?

Comment (manudeli) [reply CONTRIBUTING.md:26]: I'd like to give up stack overflow just now to maintain easily by unifying the place to ask questions

Comment (manudeli) [reply CONTRIBUTING.md:26]: I opened Discussion tab. Let's traffic to our GitHub Discussion.

Comment (manudeli) [reply CONTRIBUTING.md:26]: After remove content about stackoverflow, Let's merge this

**Review by manudeli — APPROVED** [review]
Thanks👍

**Pull request merged**